src/style.css

```diff
@@ -0,0 +1,48 @@
+.container-1 input#search {
+  width: 100%;
+  font-weight: 300;
+  float: left;
+  padding-left: 50px;
+  margin-bottom: 20px;
+  height: 70px;
+  top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0.1px 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "@Avenir";
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.container-1 .icon {
+  position: absolute;
+  box-sizing: border-box;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 45px;
+}
+
+.container-2 input#search-2 {
+  width: 100%;
+  font-weight: 300;
+  float: left;
+  padding-left: 40px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0.1px 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "@Avenir";
+  font-size: 16px;
+  line-height: 22px;
+  height: 42px;
+}
+
+.container-2 .icon-2 {
+  position: absolute;
+  left: 26px;
+  top: 127px;
+  height: 13px;
+  width: 13px;
+}
```

Comment:
check the 2nd topic in the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)